server/config/lib/app.js

```diff
@@ -6,32 +6,54 @@
 var config = require('../config'),
   mongoose = require('./mongoose'),
   express = require('./express'),
-  chalk = require('chalk'),
-  seed = require('./seed');
-
-function seedDB() {
-  if (config.seedDB && config.seedDB.seed) {
-    console.log(chalk.bold.red('Warning:  Database seeding is turned on'));
-    seed.start();
-  }
-}
-
-// Initialize Models
-mongoose.loadModels(seedDB);
+  chalk = require('chalk');
 
-module.exports.init = function init(callback) {
-  mongoose.connect(function (db) {
-    // Initialize express
-    var app = express.init(db);
-    if (callback) callback(app, db, config);
+// Establish an SQL server connection, instantiating all models and schemas
+function startSequelize() {
+  return new Promise(function (resolve, reject) {
+    let orm = null;
+    if (config.orm) {
+      orm = require('./sequelize');
+      orm.sync()
+        .then(function () {
+          resolve(orm);
+        });
+    }
+  });
+}
 
+// Establish a MongoDB connection, instantiating all models
+function startMongoose() {
+  return new Promise(function (resolve, reject) {
+    mongoose.loadModels()
+      .then(mongoose.connect)
+      .then(mongoose.seed)
+      .then(function(dbConnection) {
+        resolve(dbConnection);
+      })
   });
-};
+}
+
+/**
+ * Establish ExpressJS powered web server
+ * @param {object} db a Mongoose DB
+ * @param {object} orm an SQL DB
+ */
+function startExpress(db, orm) {
+  return new Promise(function (resolve, reject) {
+    // Initialize the ExpressJS web application server
+    const app = express.init(db, orm);
```

Comment:
Right, it's missing an actual implementation as I didn't yet create any routes/controllers the models so it's still a "mystery" until I put more code there. It is however ready to be used if you need access to any of the ORM models, the sequelizejs instance or the Sequelizejs library itself.